packages/framework/aqueduct/api-report/aqueduct.api.md

```diff
@@ -16,19 +16,20 @@ import { IContainerRuntimeBase } from '@fluidframework/runtime-definitions/inter
 import { IContainerRuntimeOptions } from '@fluidframework/container-runtime/internal';
 import { IDataStore } from '@fluidframework/runtime-definitions/internal';
 import { IEvent } from '@fluidframework/core-interfaces';
+import { IEvent as IEvent_2 } from '@fluidframework/core-interfaces/internal';
```

Comment:
duplicate import. probably need to fix some file that not using the internal path

Reply:
I was able to make that go away by splitting an internal import and moving event to a separate public import. To avoid duplication like this we have to avoid using public APIs via internal imports if they get reexports: everything works fine in that case except api rollups show duplication like this. I'm not sure its worth mitigating.